site/app/views/grading/SimpleGraderView.php

```diff
@@ -92,7 +92,8 @@ public function simpleDisplay($gradeable, $graded_gradeables, $student_full, $gr
             "sections" => $sections,
             "component_ids" => $component_ids,
             "print_lab_url" => $this->core->buildCourseUrl(['gradeable', $gradeable->getId(), 'grading', 'print']),
-            "grading_url" => $this->core->buildCourseUrl(['gradeable', $gradeable->getId(), 'grading'])
+            "grading_url" => $this->core->buildCourseUrl(['gradeable', $gradeable->getId(), 'grading']),
+            "instruction_urls" => [$gradeable->getInstructionsUrl(), $gradeable->getTaInstructions()]
```

Comment:
In my earlier review, I meant to display the instructor instruction URLs on labs just like on electronic gradeables. I didn't mention the taInstructions. Only the instructor links should be shown to students.

This should look consistent in lab gradeables and electronic gradeables so you can display the externel link with the same font awesome icon